app/src/org/commcare/dalvik/dialogs/AlertDialogFactory.java

```diff
@@ -30,6 +30,7 @@ public AlertDialogFactory(Activity context, String title, String msg) {
         messageView.setText(msg);
 
         this.dialog = builder.create();
+        dialog.setView(view);
```

Comment:
@amstone326 is there a reason this was being called in `showDialog` and not here? Seems like it should happen here in case someone tries to pull `getDialog().show()` instead of `showDialog()`.